libs/designsystem/fab-sheet/src/fab-sheet.component.ts

```diff
@@ -16,6 +16,9 @@ import { ActionSheetComponent } from '@kirbydesign/designsystem/modal';
 import { IonBackdrop, IonFab, IonFabButton, IonFabList } from '@ionic/angular/standalone';
 import { DOCUMENT } from '@angular/common';
 
+const KIRBY_FAB_SHEET_DEPRECATION_WARNING =
+  '[Kirby fab sheet] Under deprecation - Use Header Action Group instead';
```

Comment:
We should also add a similar deprecation warning to the action sheet component 😊